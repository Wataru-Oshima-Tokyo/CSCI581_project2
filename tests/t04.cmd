inp3.txt 0.1 obs4.txt 
