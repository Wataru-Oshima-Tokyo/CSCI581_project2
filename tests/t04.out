0.6715027815810661 5 
0.9916672735446840 11 
0.9875215631061130 17 
0.9859719227359411 20 
0.9873738443002035 22 
0.8943318879967835 19 
0.8022376103591111 13 
0.5578645269211857 19 
